src/components/RightPane/CoursePane/CourseRenderPane.js

```diff
@@ -228,10 +257,8 @@ class CourseRenderPane extends PureComponent {
 
             currentView = (
                 <div className={classes.root}>
-                    <div className={classes.bannerContainer}>
-                        {['COMPSCI', 'IN4MATX', 'I&C SCI', 'STATS'].includes(
-                            RightPaneStore.getFormData().deptValue
-                        ) && <RecruitmentBanner className={classes.banner} />}
+                    <div className={classes.bannerContainer} style={{ paddingRight: '0', marginBottom: '5px' }}>
```

Comment:
Now that the conditional rendering is inside the component, we can just put this wrapper div in the component as well.